pkg/forwarder/forwarder.go

```diff
@@ -152,6 +152,24 @@ type Forwarder interface {
 // Compile-time check to ensure that DefaultForwarder implements the Forwarder interface
 var _ Forwarder = &DefaultForwarder{}
 
+// Options contain the configuration options for the DefaultForwarder
+type Options struct {
+	NumberOfWorkers      int
+	RetryQueueSize       int
+	EnableHealthChecking bool
+	KeysPerDomain        map[string][]string
+}
+
+// NewOptions creates new Options with default values
+func NewOptions(keysPerDomain map[string][]string) *Options {
```

Comment:
Could you rename this function to `NewDefaultOptions` because this function didn't allow the user to set all the possible options.